validation/examples/dummy.cue

```diff
@@ -25,10 +25,13 @@
 	type:        "componen"
 }
 template: {
+	_metadata: {
+		foo: context.names
```

Comment:
Do you mean name here instead of names?

Reply:
This is the file I used for testing before I implement proper tests at some point. So just committed in its last state, where I was checking whether it catches misspellings.